Tvl.VisualStudio.JustMyCodeToggle/JustMyCodeTogglePackage.cs

```diff
@@ -1,19 +1,24 @@
 ﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
+using Microsoft;
```

Comment:
:memo: This should have been inside the namespace, but it also should have been detected by the build. I'll fix it in a follow-up pull request where I enable StyleCop Analyzers.